.flake8

```diff
@@ -6,7 +6,7 @@ exclude =
     build,
     dist,
     docs/source/conf.py
-    examples/benchmark.py,
+    examples/*.py,
```

Comment:
I'd keep it limited to `benchmark.py`, and would add a comment line to accept flake8 "violations", e.g. with `# noqa F401` (see some existing examples below in `prepare_gp_optimizer.py`).